colossus/src/main/scala/colossus/protocols/http/StreamService.scala

```diff
@@ -0,0 +1,222 @@
+package colossus
```

Comment:
Any particular reason for this style instead of `package colossus.protocols.http.stream`?